emcc

```diff
@@ -708,7 +708,7 @@ try:
   if js_opts is None: js_opts = opt_level >= 2
   if llvm_opts is None: llvm_opts = LLVM_OPT_LEVEL[opt_level]
   if opt_level == 0: debug_level = max(3, debug_level)
-  if memory_init_file is None: memory_init_file = opt_level >= 2
+  if memory_init_file is None: memory_init_file = 2 if opt_level >= 2 else 0
```

Comment:
wouldn't this change make this new mode the default in `-O2+`? But it's less optimized than a binary mem init?